api/authorization-api-1_0.md

```diff
@@ -15,6 +15,7 @@ kw:
   - Access Management
   - XACML
```

Comment:
If we're mentioning XACML and Cedar, we should mention ALFA

Reply:
done